Error in npm install

I am Reporting a bug or issue
npm install basic-setup is not working for me. I have downgraded my node version to 10.20.1 and have a sketch version of 64.
I have skpm 6.14.4 and was able to create plugin with the default skpm package.
I have also installed the latest webpack 

**Expected behavior:**
To install sketch-react using basic-setup

**Observed behavior:**
[1/2] 🖨  Copied src/manifest.json in 2ms
error Error while building ./my-command.js
./node_modules/react-sketchapp/lib/module/components/index.js 6:9
Module parse failed: Unexpected token (6:9)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
| export { Image } from './Image';
| export { RedBox } from './RedBox';
> export * as Svg from './Svg';
| export { View } from './View';
| export { Text } from './Text';
 @ ./node_modules/react-sketchapp/lib/module/index.common.js 5:0-29 5:0-29
 @ ./node_modules/react-sketchapp/lib/module/index.sketch.js
 @ ./src/my-command.js
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! basic-setup@1.0.0 build: `skpm-build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the basic-setup@1.0.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/winjingjie/.npm/_logs/2020-05-05T15_29_52_608Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! basic-setup@1.0.0 postinstall: `npm run build && skpm-link`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the basic-setup@1.0.0 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/winjingjie/.npm/_logs/2020-05-05T15_29_52_640Z-debug.log

**How to reproduce:**
`git clone https://github.com/airbnb/react-sketchapp.git
cd react-sketchapp/examples/basic-setup && npm install`

**Sketch version:**
64

**Please attach screenshots, a zip file of your project, and/or a link to your github project**

![Screenshot 2020-05-05 at 11 33 20 PM](https://user-images.githubusercontent.com/61727718/81084707-d26c5400-8f28-11ea-92b6-426cbc6f2716.png)

Sorry if this is a noob question, but in scripts : render,I can't seem to find the file 
`node ../../../skpm/packages/builder/lib/build.js`
may I know which js file this is referencing from? I have changed the directory of my node global modules to
`/home/winjingjie/.node_modules_global/bin/npm`

**The error message I am facing**

internal/modules/cjs/loader.js:638
    throw err;
    ^

`Error: Cannot find module '/Users/winjingjie/Bill/skpm/packages/builder/lib/build.js'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Function.Module.runMain (internal/modules/cjs/loader.js:831:12)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:623:3)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! basic-setup@1.0.0 render: `node ../../../skpm/packages/builder/lib/build.js --watch --run`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the basic-setup@1.0.0 render script.`

I can run build, watch and postinstall w no problems. running render:once with the --run flag takes very long to load (but that's another issue)
![Screenshot 2020-05-06 at 10 24 45 AM](https://user-images.githubusercontent.com/61727718/81133314-d24d7200-8f83-11ea-88c5-94fe168cdedd.png)

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -2,6 +2,10 @@
 
 This project adheres to [Semantic Versioning](http://semver.org/). Every release, along with the migration instructions, is documented on the Github [Releases](https://github.com/airbnb/react-sketchapp/releases) page.
 
+## Version 3.2.4
+
+- Fix the generated ES package
+
 ## Version 3.2.3
 
 - Fix getting the font name (#510)

--- a/examples/basic-setup-typescript/webpack.skpm.config.js
+++ b/examples/basic-setup-typescript/webpack.skpm.config.js
@@ -6,7 +6,7 @@ module.exports = (config) => {
       ...config.resolve,
       alias: {
         ...config.resolve.alias,
-        'react-sketchapp': path.resolve(__dirname, '../../lib/'),
+        'react-sketchapp': path.resolve(__dirname, '../../'),
       },
     };
   }

--- a/examples/basic-setup/webpack.skpm.config.js
+++ b/examples/basic-setup/webpack.skpm.config.js
@@ -6,7 +6,7 @@ module.exports = (config) => {
       ...config.resolve,
       alias: {
         ...config.resolve.alias,
-        'react-sketchapp': path.resolve(__dirname, '../../lib/'), // FIXME: Should be able to do '../../', as that's a close simulation of an 'npm install' and import 'react-sketchapp'
+        'react-sketchapp': path.resolve(__dirname, '../../'),
       },
     };
   }

--- a/examples/basic-svg/webpack.skpm.config.js
+++ b/examples/basic-svg/webpack.skpm.config.js
@@ -6,7 +6,7 @@ module.exports = (config) => {
       ...config.resolve,
       alias: {
         ...config.resolve.alias,
-        'react-sketchapp': path.resolve(__dirname, '../../lib/'),
+        'react-sketchapp': path.resolve(__dirname, '../../'),
       },
     };
   }

--- a/examples/colors/webpack.skpm.config.js
+++ b/examples/colors/webpack.skpm.config.js
@@ -6,7 +6,7 @@ module.exports = (config) => {
       ...config.resolve,
       alias: {
         ...config.resolve.alias,
-        'react-sketchapp': path.resolve(__dirname, '../../lib/'),
+        'react-sketchapp': path.resolve(__dirname, '../../'),
       },
     };
   }

--- a/examples/emotion/webpack.skpm.config.js
+++ b/examples/emotion/webpack.skpm.config.js
@@ -6,7 +6,7 @@ module.exports = (config) => {
       ...config.resolve,
       alias: {
         ...config.resolve.alias,
-        'react-sketchapp': path.resolve(__dirname, '../../lib/'),
+        'react-sketchapp': path.resolve(__dirname, '../../'),
       },
     };
   }

--- a/examples/form-validation/webpack.skpm.config.js
+++ b/examples/form-validation/webpack.skpm.config.js
@@ -6,7 +6,7 @@ module.exports = (config) => {
       ...config.resolve,
       alias: {
         ...config.resolve.alias,
-        'react-sketchapp': path.resolve(__dirname, '../../lib/'),
+        'react-sketchapp': path.resolve(__dirname, '../../'),
       },
     };
   }

--- a/examples/foursquare-maps/webpack.skpm.config.js
+++ b/examples/foursquare-maps/webpack.skpm.config.js
@@ -6,7 +6,7 @@ module.exports = (config) => {
       ...config.resolve,
       alias: {
         ...config.resolve.alias,
-        'react-sketchapp': path.resolve(__dirname, '../../lib/'),
+        'react-sketchapp': path.resolve(__dirname, '../../'),
       },
     };
   }

--- a/examples/glamorous/webpack.skpm.config.js
+++ b/examples/glamorous/webpack.skpm.config.js
@@ -6,7 +6,7 @@ module.exports = (config) => {
       ...config.resolve,
       alias: {
         ...config.resolve.alias,
-        'react-sketchapp': path.resolve(__dirname, '../../lib/'),
+        'react-sketchapp': path.resolve(__dirname, '../../'),
       },
     };
   }

--- a/examples/profile-cards-graphql/webpack.skpm.config.js
+++ b/examples/profile-cards-graphql/webpack.skpm.config.js
@@ -6,7 +6,7 @@ module.exports = (config) => {
       ...config.resolve,
       alias: {
         ...config.resolve.alias,
-        'react-sketchapp': path.resolve(__dirname, '../../lib/'),
+        'react-sketchapp': path.resolve(__dirname, '../../'),
       },
     };
   }

--- a/examples/profile-cards-primitives/webpack.skpm.config.js
+++ b/examples/profile-cards-primitives/webpack.skpm.config.js
@@ -6,7 +6,7 @@ module.exports = (config) => {
       ...config.resolve,
       alias: {
         ...config.resolve.alias,
-        'react-sketchapp': path.resolve(__dirname, '../../lib/'),
+        'react-sketchapp': path.resolve(__dirname, '../../'),
       },
     };
   }

--- a/examples/profile-cards-react-with-styles/webpack.skpm.config.js
+++ b/examples/profile-cards-react-with-styles/webpack.skpm.config.js
@@ -6,7 +6,7 @@ module.exports = (config) => {
       ...config.resolve,
       alias: {
         ...config.resolve.alias,
-        'react-sketchapp': path.resolve(__dirname, '../../lib/'),
+        'react-sketchapp': path.resolve(__dirname, '../../'),
       },
     };
   }

--- a/examples/profile-cards/webpack.skpm.config.js
+++ b/examples/profile-cards/webpack.skpm.config.js
@@ -6,7 +6,7 @@ module.exports = (config) => {
       ...config.resolve,
       alias: {
         ...config.resolve.alias,
-        'react-sketchapp': path.resolve(__dirname, '../../lib/'),
+        'react-sketchapp': path.resolve(__dirname, '../../'),
       },
     };
   }

--- a/examples/react-router-prototyping/webpack.skpm.config.js
+++ b/examples/react-router-prototyping/webpack.skpm.config.js
@@ -6,7 +6,7 @@ module.exports = (config) => {
       ...config.resolve,
       alias: {
         ...config.resolve.alias,
-        'react-sketchapp': path.resolve(__dirname, '../../lib/'),
+        'react-sketchapp': path.resolve(__dirname, '../../'),
       },
     };
   }

--- a/examples/styled-components/webpack.skpm.config.js
+++ b/examples/styled-components/webpack.skpm.config.js
@@ -6,7 +6,7 @@ module.exports = (config) => {
       ...config.resolve,
       alias: {
         ...config.resolve.alias,
-        'react-sketchapp': path.resolve(__dirname, '../../lib/'),
+        'react-sketchapp': path.resolve(__dirname, '../../'),
       },
     };
   }

--- a/examples/styleguide/webpack.skpm.config.js
+++ b/examples/styleguide/webpack.skpm.config.js
@@ -6,7 +6,7 @@ module.exports = (config) => {
       ...config.resolve,
       alias: {
         ...config.resolve.alias,
-        'react-sketchapp': path.resolve(__dirname, '../../lib/'),
+        'react-sketchapp': path.resolve(__dirname, '../../'),
       },
     };
   }

--- a/examples/symbols/webpack.skpm.config.js
+++ b/examples/symbols/webpack.skpm.config.js
@@ -6,7 +6,7 @@ module.exports = (config) => {
       ...config.resolve,
       alias: {
         ...config.resolve.alias,
-        'react-sketchapp': path.resolve(__dirname, '../../lib/'),
+        'react-sketchapp': path.resolve(__dirname, '../../'),
       },
     };
   }

--- a/examples/timeline-airtable/webpack.skpm.config.js
+++ b/examples/timeline-airtable/webpack.skpm.config.js
@@ -6,7 +6,7 @@ module.exports = (config) => {
       ...config.resolve,
       alias: {
         ...config.resolve.alias,
-        'react-sketchapp': path.resolve(__dirname, '../../lib/'),
+        'react-sketchapp': path.resolve(__dirname, '../../'),
       },
     };
   }

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "react-sketchapp",
-  "version": "3.2.3",
+  "version": "3.2.4",
   "description": "A React renderer for Sketch.app",
   "sideEffects": false,
   "main": "lib/index.js",

--- a/src/components/index.ts
+++ b/src/components/index.ts
@@ -3,6 +3,7 @@ export { Page } from './Page';
 export { Artboard } from './Artboard';
 export { Image } from './Image';
 export { RedBox } from './RedBox';
-export * as Svg from './Svg';
 export { View } from './View';
 export { Text } from './Text';
+import * as Svg from './Svg';
+export { Svg };

